spec/forms/candidate_interface/becoming_a_teacher_form_spec.rb

```diff
@@ -12,6 +12,8 @@
       becoming_a_teacher: data[:becoming_a_teacher],
     }
   end
+  let(:application_form) { create(:application_form) }
+  let(:becoming_a_teacher) { described_class.new(form_data) }
```

Comment:
```suggestion
  subject(:becoming_a_teacher) { described_class.new(form_data) }
```
NAB - i normally conventionally use subject for the main subject of the test